api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -46,6 +51,14 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        if (personService.getPersonAttributeTypeByName("accessControlLocation") == null) {
```

Comment:
I thought you would have a global property which points to a uuid of the person attribute type to store the location.

Reply:
Oh, So I need to check and create a global property when the module starts like we creating the person attribute. then after, I can use that global property to get the Attribute Name. 
Global property Name : accessControlLocationUuid
Global property value : that uuid 

Is it correct @dkayiwa ?

Reply:
locationbasedaccess.locationAttributeUuid

Reply:
I have updated my code to retrieve the uuid from the global property. Please have a look again @dkayiwa